app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str):
+    names = command.split('')
```

Comment:
your separator in split method is empty. Just delete any quotes, split() use space by default